execution_engine/src/storage/trie_store/operations/mod.rs

```diff
@@ -1100,7 +1065,7 @@ where
                             maybe_next_trie = {
                                 match self.store.get_raw(self.txn, pointer.hash()) {
                                     Ok(Some(trie_bytes)) => {
-                                        match trie::lazy_trie_deserialize(trie_bytes) {
+                                        match bytesrepr::deserialize(trie_bytes.into()) {
```

Comment:
```suggestion
                                        match bytesrepr::deserialize_from_slice(&trie_bytes) {
```

Reply:
fixed in https://github.com/casper-network/casper-node/pull/4083/commits/42c4d61a9a2fea9d299d65b128d3854d869f1870